cmd/internal/root.go

```diff
@@ -26,7 +27,7 @@ import (
 var log *zap.SugaredLogger
 
 func wrapError(err error, msg string) error {
-	return errors.Wrap(err, "quarks-job command failed. "+msg)
+	return errors.Wrap(err, fmt.Sprintf("quarks-job command failed. %s", msg))
```

Comment:
nit: `Wrapf` instead?

Reply:
right! addressed in https://github.com/cloudfoundry-incubator/quarks-job/pull/80/commits/d691c0bfff0960c1fc7a97ef96536929e109640f :slightly_smiling_face: